.eslintrc

```diff
@@ -30,7 +30,7 @@
     "no-shadow": 2,
     "no-shadow-restricted-names": 2,
     "no-spaced-func": 2,
-    "no-trailing-spaces": 2,
+    "no-trailing-spaces": 0,
```

Comment:
On the other hand, maybe it's just me being too impulsive at the end of the hard day. I've not decided yet, tbh 😄